src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/AbstractClassCouplingCheck.java

```diff
@@ -73,6 +75,12 @@
     private int max;
     /** Package of the file we check. */
     private String packageName;
+    /** User-configured regular expression to ignore classes. */
+    private String excludeClassesRegexp = "";
```

Comment:
Can't we just left it empty? I have condition on it inside class (excludeClassesRegexp.isEmpty())